runtime/vm/VMAccess.cpp

```diff
@@ -837,6 +842,10 @@ releaseExclusiveVMAccessFromExternalThread(J9JavaVM * vm)
 {
 	J9VMThread * currentThread;
 	Assert_VM_true(J9_XACCESS_EXCLUSIVE == vm->exclusiveAccessState);
```

Comment:
>  but not if safePoint is acquired (since it does not set `exclusiveAccessState` to `J9_XACCESS_EXCLUSIVE`).

There might be a chance to trigger this assertion based on the above comment. Should we match #18535 and place the assertion after the below `if` block?

Reply:
Coorrect. Will update.